engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/bulk-actions-environment.html

```diff
@@ -41,7 +41,7 @@ <h4 translate>Assign Environment and Content View</h4>
 
       <button class="btn btn-default fr"
               translate
-              ng-hide="denied('edit_content_hosts', contentHost)"
+              ng-hide="denied('edit_hosts', contentHost)"
```

Comment:
With these bulk actions, there isn't even a $scope.contentHost, so i think that is just passing in nil.